### PR TITLE
Introduce Sequelize

### PR DESCRIPTION
**Pull request by rrodionov91**

### CODE REVIEW

File: server/controllers/user.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import { dataUser } from "../Base/dataUser.js"
-import { client } from "../loaders/client.js"
+import { sequelize } from "../loaders/client.js"
 
 export class UserController {
     async profile (req, res) {
         if(req.session.userLogin !== null){
-          let result = await dataUser(client, req.session.userLogin)
+          let result = await dataUser(sequelize, req.session.userLogin)
```
Comment (rrodionov91) [inline server/controllers/user.js:7]: Please use UserService unstead of plain SQL

File: server/services/TaskService.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import {sequelize} from "../loaders/client.js";
+
+export class TaskService {
+    constructor(){
+        this.taskModule = sequelize.models.Task
```
Comment (rrodionov91) [inline server/services/TaskService.js:5]: taskModel instead of taskModule

File: server/services/TaskService.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    }
+
+    async getTask(userLogin) {
```
Comment (rrodionov91) [inline server/services/TaskService.js:8]: getTasksByExecutor

File: server/services/TaskService.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+            where: {executor: userLogin}
+            })
+            return result
+            
+        } catch (error){
+            console.error(`"Error, failed to get tasks by login" ${error}`)
+        }
+    }
+
+    async creatingTask({ executor, author, title, content, status, deadline, creationDate, completeDate }) {
```
Comment (rrodionov91) [inline server/services/TaskService.js:20]: createTask

File: server/services/TaskService.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    }
+
+    async updateTask({ taskId, executors, author, title, content, status, deadline, completeDate }) {
+        try {
+            await this.taskModule.update(
+                {executors, author, title, content, status, deadline, completeDate},
+                {where:{id: taskId},}
+            ) 
+        } catch(err){
+            console.error(err)
```
Comment (rrodionov91) [inline server/services/TaskService.js:37]: Please add message to log same as in createTask method

File: server/services/UserService.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
+import {sequelize} from "../loaders/client.js";
+
+export class UserService {
+    constructor() {
+        this.userModel = sequelize.models.User
+    }
+    
+    async createUser({ firstName, lastName, login, email, password }) {
+        try {
+            await this.userModel.create({ // if you use "this.UserModel.create", he swears that he cannot read the "create" property of the "UserModel" object
```
Comment (rrodionov91) [inline server/services/UserService.js:10]: Please delete comment

File: server/services/UserService.js [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+        } catch (err) {
+            console.log(err)
+        }
+    }
+
+    async getUser(loginReg) {
+        try {
+            let result = await this.userModel.findAll({
+                attributes: ['login', 'password'],
+                where: {login: `${loginReg}`}
```
Comment (rrodionov91) [inline server/services/UserService.js]: Strange string template

**Review by studentmak — APPROVED** [review]

**Pull request merged**